python/lsst/ts/cbp/component.py

```diff
@@ -0,0 +1,151 @@
+import socket
+from time import sleep
+
+
+class CBPComponent:
```

Comment:
Consider adding a doc string here. Something like below. However it's more important to follow standards to maintain consistency with what Systems engineering does. 

"""stuff that does stuff

    Parameters
    ----------
    foo : `taco.Bar`
        An object that does X, Y and Z
"""